INTERPRETING.md

```diff
@@ -0,0 +1,175 @@
+# Interpreting Test262 Tests
+
+All tests are declared as text files located within this project's `test`
+directory. In order to execute Test262 tests, runtimes must observe the
+following semantics.
+
+## Test Execution
+
+Test262 tests are only valid under the runtime environment conditions described
+here. Test environments may be further modified according to the metadata
+contained with each test--refer to the "Metadata" section for more details.
+
+### Realm Isolation
+
+Each test must be executed in a new [ECMAScript
+realm](https://tc39.github.io/ecma262/#sec-code-realms) dedicated to that test.
+Unless configured otherwise (via the `module` flag), source text must be
+interpreted as [global
+code](http://www.ecma-international.org/ecma-262/6.0/#sec-types-of-source-code).
+
+### Test262-Defined Bindings
+
+The contents of the following files must be evaluated in the test realm's
+global scope prior to test execution:
+
+1. `harness/assert.js`
+2. `harness/sta.js`
+
+### Host-Defined Functions
+
+The following values must be defined as writable, configurable, non-enumerable
+properties of the global scope prior to test execution.
+
+- **`print`** A function that exposes the string value of its first argument to
+  the test runner. This is used as a communication mechanism for asynchronous
+  tests (via the `async` flag, described below).
+
+### Strict Mode
+
+Unless configured otherwise (via the `noStrict`, `onlyStrict`, or `raw` flags),
+each test must be executed twice: once in ECMAScript's non-strict mode, and
+again in ECMAScript's strict mode.
+
+## Test Results
+
+By default, tests signal failure by generating an uncaught exception. If
+execution completes without generating an exception, the test must be
+interpreted as "passing." Any uncaught exception must be interpreted as test
+failure. These semantics may be modified by any test according to the metadata
+declared within the test itself (via the `negative` attribute and the `async`
+flag, described below).
+
+## Metadata
+
+Each test file may define metadata that describe additional requirements. This
+information is delimited by the token sequence `/*---` and `---*/` and is
+structured as [YAML](http://yaml.org/).
```

Comment:
Sure! I'm not really familiar with YAML, but from reading the spec, I believe this is a reasonably complete description of the subset we use:

> [...] structured using a subset of [YAML](http://yaml.org/):
>
> - Block mappings
> - Flow scalars (including both single- and double- quoted styles)
> - Flow sequences
> - Block sequences
> - Block scalars
>
> For example:
>
> ```yaml
> first: scalar value
> second: "`quoted` scalar value"
> third: [a, flow, sequence]
> fourth:
>    - a
>    - block
>    - sequence
> fourth: |
>     A block scalar (without newline folding)
> fifth: >
>     Another block scalar (with newline folding)
> ```

Does this look good to you? (By the way, this will have an impact on the design of gh-382.)